### ISSUE TITLE
How would you like to save dialog: accessibility issue.

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#6209]
### Steps to reproduce

With NVDA screen reader running, if the How would you like to Save? dialog opens, then NVDA reads all the following text:
![image](https://github.com/audacity/audacity/assets/11887021/44afd442-8c50-48b6-9f7f-3a3adb8d9abb)

### Expected behavior

There should be a pause between the sentence "You can access your work..." and the heading "On your computer"

### Actual behavior

There isn't a pause, because there isn't a full stop at the end of the sentence. This makes the meaning very confusing.

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

### Additional context

This isn't an issue using either Jaws or Narrator, as the text on the right hand side of the dialog is not read.
